test/e2e/azure_test.go

```diff
@@ -96,6 +96,14 @@ var _ = Describe("Workload cluster creation", func() {
 				WaitForControlPlaneIntervals: e2eConfig.GetIntervals(specName, "wait-control-plane"),
 				WaitForMachineDeployments:    e2eConfig.GetIntervals(specName, "wait-worker-nodes"),
 			})
+
+			Context("in a cluster with a single node", func() {
```

Comment:
I don't think there is much value in this test with a single node in general, so I would prefer to remove this test and not add to it